Sample/src/androidTest/java/dev/testify/sample/compose/ComposableScreenshotTest.kt

```diff
@@ -157,16 +157,16 @@ class ComposableScreenshotTest {
         val locale = Locale.JAPAN
         val fontScale = 3.0f
         rule
+            .configure {
```

Comment:
Without this change, you couldn't call `configure` here as it would return a rule instance cast as the base `ScreenshotRule` instead of the specific `ComposableScreenshotRule`. By returning `ComposableScreenshotRule`, you can now successfully chain other methods exposed by that class.